ci-operator/step-registry/ipi/conf/aws/rootvolume/ipi-conf-aws-rootvolume-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+if [[ "${AWS_COMPUTE_NODE_VOLUME_TYPE}" != "" ]]; then
```

Comment:
```suggestion
if [[ "${AWS_COMPUTE_VOLUME_TYPE}" != "" ]]; then
```

Reply:
How about also introduce `AZURE_CONTROL_PLANE_VOLUME_TYPE` and `AZURE_DEFAULT_MACHINE_VOLUME_TYPE` to make this step to be more powerful.

Reply:
size must be provided if type is specified.

Reply:
good to know.